docs/repositories-configure/managing-branches.md

```diff
@@ -1,9 +1,9 @@
 # Managing branches
 
 {% if not extra.self_hosted %}
-Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
+Just like on your Git provider, every repository on Codacy has a default branch. Codacy always analyzes the default branch and loads its data first on dashboards. When you add a repository, the default branch on Codacy matches the default branch on your Git provider (typically `master` or `main`) and is the only one being analyzed.
```

Comment:
Yup, had the same feeling. I reverted leaving a minor edit: a mention that Codacy loads the default branch first (that was our whole point in the end). Let me know if it works like this.

```suggestion
Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider) and loads its data first on dashboards. Codacy also supports analyzing multiple branches.
```

Reply:
@prcr
Let me know if this is better (it opened this new conversation for some reason)

Reply:
Yes, it's more straight to the point now. :smile: Nice editing!

Reply:
@nicklem I think you forgot to apply the changes that you were suggesting in this thread. :slightly_smiling_face:

Reply:
@prcr Good catch! Just opened https://github.com/codacy/docs/pull/1583 to fix